lib/travis/api/v3/renderer/user_builds.rb

```diff
@@ -0,0 +1,6 @@
+module Travis::API::V3
+  class Renderer::UserBuilds < CollectionRenderer
```

Comment:
I think this should just use the builds resource. User builds as its own resource type don't make a lot of sense in my opinion.